packages/snaps-rpc-methods/src/restricted/dialog.test.tsx

```diff
@@ -446,7 +495,7 @@ describe('implementation', () => {
           params: {} as any,
         }),
       ).rejects.toThrow(
-        'Invalid params: Expected the value to satisfy a union of `object | object`, but received: [object Object]',
+        /Invalid params: At path: .* -- Expected type to be one of: .*, but received: .*/u,
```

Comment:
Can we not test without regex here?

Reply:
Potentially, but then we need to update it every time we add a component.